doc/source/user_guide/options.rst

```diff
@@ -31,7 +31,7 @@ namespace:
 * :func:`~pandas.option_context` - execute a codeblock with a set of options
   that revert to prior settings after execution.
 
-**Note:** Developers can check out `pandas/core/config.py <https://github.com/pandas-dev/pandas/blob/master/pandas/core/config.py>`_ for more information.
+**Note:** Developers can check out `pandas/core/config.py <https://github.com/pandas-dev/pandas/blob/master/pandas/core/config_init.py>`_ for more information.
```

Comment:
Should also be `pandas/core/config_init.py` ? https://github.com/pandas-dev/pandas/blob/master/pandas/core/config_init.py

Reply:
@enryH can you update this as well

Reply:
So [`core/config_init.py`](https://github.com/pandas-dev/pandas/blob/master/pandas/core/config_init.py) is the link I updated the former link to. Do you also want to link [_config/config.py](https://github.com/pandas-dev/pandas/blob/master/pandas/_config/config.py)?

Reply:
ahh its the *title* of the link that needs fixing here.

Reply:
I changed it to `config_init.py`